src/sagemaker/jumpstart/factory/utils.py

```diff
@@ -0,0 +1,67 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module stores JumpStart factory utilities."""
+
+from __future__ import absolute_import
+from typing import Union
+
+from sagemaker.jumpstart.constants import DEFAULT_JUMPSTART_SAGEMAKER_SESSION
+from sagemaker.jumpstart.types import (
+    JumpStartEstimatorDeployKwargs,
+    JumpStartEstimatorFitKwargs,
+    JumpStartEstimatorInitKwargs,
+    JumpStartModelDeployKwargs,
+    JumpStartModelInitKwargs,
+)
+
+KwargsType = Union[
+    JumpStartModelDeployKwargs,
+    JumpStartModelInitKwargs,
+    JumpStartEstimatorFitKwargs,
+    JumpStartEstimatorInitKwargs,
+    JumpStartEstimatorDeployKwargs,
+]
+
+
+def get_model_info_kwargs(
```

Comment:
How about `get_model_info_default_kwargs`, potentially there are other kwargs but these are passed by default to each calls